docs/usage/sat_bootprep.md

```diff
@@ -18,7 +18,7 @@ install, upgrade, and deploy products on systems managed by CSM. Outside of IUF,
 it is uncommon to use `sat bootprep`. For more information on this relationship,
 see [SAT and IUF](sat_and_iuf.md). For more information on IUF, see the
 [IUF section](https://cray-hpe.github.io/docs-csm/en-14/operations/iuf/iuf/) of
-the [*Cray System Management Documentation*](https://cray-hpe.github.io/docs-csm/).
+the [*Cray System Management Documentation*](https://github.com/Cray-HPE/docs-csm/blob/release/1.5/operations/configuration_management/Accessing_Sat_Bootprep_Files.md#accessing-sat-bootprep-files).
```

Comment:
I'm not sure why you are changing this link. This link was intended to be a top-level link to the HTML version of CSM documentation, and from there, the reader should be able to find the section named "IUF section". Can you please explain what you are trying to achieve?

Reply:
I will also add some rationale for why we used these top-level links to the CSM documentation. As you'll notice in the link immediately preceding this one, if you want to link to a specific section of the CSM documentation, you have to include the CSM documentation version (e.g. "en-14"). As a result, it was hard for us to keep those links up-to-date, so we opted to link to the top level of the documentation without a version. When visiting this link, it will automatically default to the latest version of the CSM documentation.

With the move of docs-sat content into docs-csm (see my top-level comment), linking to other sections of docs-csm from the SAT documentation is now easier because we can use relative links to other markdown files within the same repository. So perhaps the problem you're trying to address is already fixed in the SAT documentation in docs-csm.

Reply:
Hello @haasken-hpe 
Thanks for the detailed notes. It does cover the update I requested.
I agree that the above flow is more useful.
I verified that in CSM-1.6 branch, the update is already exists.
I will close this pull request.
Thanks.